### ISSUE TITLE
Serial: no member named 'RDR'; did you mean 'DR'? (STM32F411E-DISCO)

### ISSUE THREAD

**Issue by optimizedsms** [marangisto/hal#1]
There is probably something I have not done correctly, or not read...but when I try to add the usart to my code I get these errors.  (STM32F411E-DISCO)
Thank you, Mike

../../../../hal/include/usart.h: In static member function 'static void hal::usart::usart_t<NO, TX, RX>::write(uint8_t)':
../../../../hal/include/usart.h:90:16: error: 'struct stm32f411::usart2_t' has no member named 'TDR'; did you mean 'DR'?
         USART2.TDR = x;
                ^~~
../../../../hal/include/usart.h: In static member function 'static void hal::usart::usart_t<NO, TX, RX>::isr()':
../../../../hal/include/usart.h:110:26: error: 'struct stm32f411::usart2_t' has no member named 'RDR'; did you mean 'DR'?
         fifo::put(USART2.RDR);

**Comment by marangisto** [comment]
Hi, I'm sorry I didn't notice your issue earlier. I have now added support for USART on the STM32F411, something I had skipped previously as the DISCOVERY board doesn't seem to support a virtual COM port. I do see transmit output on my scope though so it might be in working condition now. When I get the chance I will test with an FTDI adapter. In any case, I added a Serial example in the STM32F411E-DISCO subdirectory and made sure it builds.
